recipe/meta.yaml

```diff
@@ -38,7 +38,8 @@ requirements:
     - libtheora
     - libnetcdf 4.4.*
     - pugixml
-    - mesa
+    - mesalib
+    - gcc  # [linux]
```

Comment:
gcc is never needed on linux, just sometimes in osx for fortran support

Reply:
When you have a dependency built with the `conda` package, the dependent package should be too.